src/pages/security.js

```diff
@@ -230,7 +240,7 @@ const SecurityPage = () => (
           </ul>
 
           <p>
-            We encourage (but don’t require) you to encrypt the information, with our [PGP key].
+            We encourage (but don’t require) you to encrypt the information, with our <ExternalLink href="/security_at_centrifuge_io.asc">PGP Key</ExternalLink>.
```

Comment:
```suggestion
            We encourage (but don’t require) you to encrypt the information, with our PGP Key <ExternalLink href="/security_at_centrifuge_io.asc">[D5B96F0E091A0ABE2A7DF6200A646339E40C2151]</ExternalLink>.
```
I've signed the key with my key and uploaded the key to a keyserver so it will be easier to find/download as well. The key can now just be downloaded in gnupg with `gpg --receive-keys D5B96F0E091A0ABE2A7DF6200A646339E40C2151` instead of downloading the .asc file.